digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala

```diff
@@ -448,7 +448,7 @@ class ProjectServiceSpec  extends FunSuite with Matchers with BeforeAndAfter {
       val projectLinks = ProjectDAO.fetchByProjectNewRoadPart(5, 205, saved.id)
       count = countCurrentProjects.size + 1
       countAfterInsertProjects.size should be(count)
-      sqlu"""UPDATE Project_link set status = 1""".execute
+      sqlu"""UPDATE Project_link set status = 5""".execute
```

Comment:
Use enumeration here, exactly for this reason
replace 5 with ${LinkStatus.Terminated.value}